content/posts/2020-05-13/index.md

```diff
@@ -0,0 +1,14 @@
+---
+title: Building, authenticating and hosting VueJS App with AWS Amplify
+description: "In this project, we will setup a brand new VueJS app, use your own AWS account, and add Vue CLI and AWS Amplify CLI via your favorite terminal."
+banner: "./banner.png"
+authorIds:
+  - bryan-chua
+href: https://medium.com/@ykbryan/building-authenticating-and-hosting-vuejs-app-with-aws-amplify-7285b7a8e90c
+platforms:
+  - Vue
+categories:
+  - Hosting
+  - Authentication
+---
+
```

Comment:
Not 100% sure but I think that this new line at the end of the file could be causing the build to fail.

Reply:
@dabit3 ok removed.